### ISSUE TITLE
Support for `windows/arm64` or `windows/aarch64`

### ISSUE THREAD

**Issue by gsmachado** [elastic/golang-crossbuild#258]
First of all, this repo is awesome!

How can I help you guys to add support for `windows/arm64` or `windows/aarch64`?

Can you give me some direction?

**Comment by iamkhalidbashir** [comment]
bump

**Comment by cmacknz** [comment]
We have had several requests to support Elastic Agent and other products on windows/arm64 but I don't think we've prioritized doing that yet, which is why this architecture is missing here.

I think we'd probably want to add a windows/arm64 machine to the CI system here, which you can't do as an external contribution. If this could be implemented by cross-compiling from windows/amd64 to windows/arm64 you wouldn't need that, but I haven't looked into how feasible that is.

**Comment by cmaglie** [comment]
It seems that Windows/arm64 is now available? Can this issue be closed?

**Comment by cmacknz** [comment]
Yes, support for windows/arm was added in https://github.com/elastic/golang-crossbuild/pull/652 and there is now a technical preview of Elastic Agent on windows/arm available to download.
